scripts/migrate

```diff
@@ -60,5 +60,5 @@ fi
 
 gcloud builds submit "${ROOT}" \
   --config "${ROOT}/builders/migrate.yaml" \
-  --substitutions "_DB_CONN=${DB_CONN},_DB_APIKEY_DATABASE_KEY=${DB_APIKEY_DATABASE_KEY},_DB_APIKEY_SIGNATURE_KEY=${DB_APIKEY_SIGNATURE_KEY},_DB_ENCRYPTION_KEY=${DB_ENCRYPTION_KEY},_DB_NAME=${DB_NAME},_DB_PASSWORD=${DB_PASSWORD},_DB_USER=${DB_USER}" \
+  --substitutions "_DB_CONN=${DB_CONN},_DB_APIKEY_DATABASE_KEY=${DB_APIKEY_DATABASE_KEY},_DB_APIKEY_SIGNATURE_KEY=${DB_APIKEY_SIGNATURE_KEY},_DB_ENCRYPTION_KEY=${DB_ENCRYPTION_KEY},_DB_NAME=${DB_NAME},_DB_PASSWORD=${DB_PASSWORD},_DB_USER=${DB_USER},_DB_DEBUG=${DB_DEBUG},_LOG_DEBUG=${LOG_DEBUG}" \
```

Comment:
I really wish Cloud Build let you specify --substitutions multiple times to avoid this insanely long line...